toontown/spellbook/MagicWordIndex.py

```diff
@@ -335,7 +335,7 @@ def handleWord(self, invoker, avId, toon, *args):
         mgId = None
         mgDiff = None if difficulty == 0 else difficulty
         mgKeep = None
-        mgSzId = ZoneUtil.getSafeZoneId(toon.zoneId) if isTeleport else None
+        mgSzId = ZoneUtil.getSafeZoneId(toon.zoneId)
```

Comment:
When requesting another game after finishing a game (on the purchase screen), `mgSzId` gets set to a dynamic zone instead of the playground's, which will crash either the game or AI when loading a game.

Reply:
Patched that up in 5814facc54bba9d334a7daf0b5ea315b08cd83b2.